modules/eloqua_api_auth_fallback/config/schema/eloqua_api_auth_fallback.schema.yml

```diff
@@ -0,0 +1,13 @@
+eloqua_api_auth_fallback.settings:
+  type: config_object
+  label: 'Eloqua API Authentication settings'
```

Comment:
can we name this to something a little more descriptive? maybe:
"**Eloqua API resource owner password credentials**" or something reflecting this?

Reply:
Yes. I have updated the code to incorporate this change.